test/other/metadce/test_metadce_minimal_64.jssize

```diff
@@ -1 +1 @@
-6203
+3844
```

Comment:
I would hope this size would not increase.. unless this test is running in MAXIMUM_MEMORY > FOUR_GB mode?  (which I don't think it is is it?)